edweiss-firebase/functions/src/functions/lectures/likeQuestion.ts

```diff
@@ -3,7 +3,7 @@ import { onAuthentifiedCall } from 'utils/firebase';
 import { clean, CollectionOf, getDocumentAndRef } from 'utils/firestore';
 import { fail, ok } from 'utils/status';
 
-export const updateQuestion = onAuthentifiedCall(LectureDisplay.Functions.likeQuestion, async (userId, args) => {
+export const likeQuestion = onAuthentifiedCall(LectureDisplay.Functions.likeQuestion, async (userId, args) => {
```

Comment:
Yes thank you this was my mistake

Reply:
No big deal, we tackle it as a great team work.